services/src/main/java/org/keycloak/forms/login/freemarker/model/UrlBean.java

```diff
@@ -93,6 +93,10 @@ public String getFirstBrokerLoginUrl() {
         return Urls.firstBrokerLoginProcessor(baseURI, realm).toString();
     }
 
+    public String getResourcesUrl() {
+        return getHostPortScheme(baseURI)+ getResourcesPath();
```

Comment:
The following should work and is much cleaner than manually building the hostname bit:

`Urls.themeRoot(baseURI).toString() + "/" + theme.getType().toString().toLowerCase() +"/" + theme.getName()`